src/main/java/com/javabom/cafe/controller/OrderController.java

```diff
@@ -0,0 +1,24 @@
+package com.javabom.cafe.controller;
+
+import com.javabom.cafe.controller.dto.NewOrderDto;
+import com.javabom.cafe.controller.dto.OrdersDto;
+import com.javabom.cafe.service.OrderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class OrderController {
+
+    @Autowired
+    private OrderService orderService;
+
+    @RequestMapping(value = "/order", method = RequestMethod.POST)
```

Comment:
RestFul하게 URI를 작성할 때는 자원을 복수형으로 작성해 주시는 것이 좋습니다!

```suggestion
    @RequestMapping(value = "/orders", method = RequestMethod.POST)
```

[NHN의 글](https://meetup.toast.com/posts/92)이 잘 정리가 되어 있다고 생각이 되서 링크 드릴게요!